docs/extensibility/theme-support.md

```diff
@@ -151,3 +151,11 @@ body.gutenberg-editor-page .editor-block-list__block[data-align="full"] {
 You can use those editor widths to match those in your theme. You can use any CSS width unit, including `%` or `px`.
 
 Further reading: [Applying Styles with Stylesheets](https://wordpress.org/gutenberg/handbook/blocks/applying-styles-with-stylesheets/).
+
+## Styling blocks
+
+Core blocks come with default styles. The styles are always enqueued for the editor but, by default, are not enqueued for the front end. If you'd like to take advantage of these styles on the front end, simply add theme support for `wp-block-styles`, and the styles will be enqueued.
```

Comment:
I think we can tweak this and cut it down to improve readability:

```markdown
**Core blocks include default styles.** The styles are enqueued for the editor but are not enqueued for a theme unless the theme opts-in to the core styles. If you'd like to use default styles in your theme, add theme support for `wp-block-styles`:
```

If I'm wrong about "theme" support versus "front-end" support let me know, but I think that's what these docs are trying to say.

Reply:
Thanks, @tofumatt. That is definitely an improvement. I made some small adjustments to the text above based on my understanding. I felt like "theme" was too general in "not enqueued for a theme" since the theme is always a factor, even in editing. What do you think of the latest?